lang/en.json

```diff
@@ -1513,7 +1513,7 @@
ct jointly.</p><p class=\"govuk-body\">When there are replacement attorneys, if an original attorney dies or can no longer act:</p><ul class=\"govuk-list govuk-list--bullet\"><li>all the replacement attorneys will replace all the original attorneys</li><li>the remaining original attorneys will not be able to make any decisions on the donor’s behalf anymore</li></ul><h3 class=\"govuk-heading-m\">Examples of restrictions and conditions to change what happens when a joint attorney dies or is unable to act</h3><p class=\"govuk-body\">If the original attorneys are appointed jointly, the donor can add restrictions and conditions to change what happens when an original attorney dies or is unable or unwilling to act.</p><p class=\"govuk-body\">Example:</p><div class=\"govuk-inset-text\"><p class=\"govuk-body\">‘If one of my original joint attorneys, Jane Doe or John Doe, is unable or unwilling to act, I then reappoint my remaining original attorney(s), Jane Doe or John Doe, as a replacement attorney.’</p></div><h2 class=\"govuk-heading-l\">Replacing attorneys who act jointly for some decisions</h2><p class=\"govuk-body\">Replacement attorneys are an important backup when <a href=\"{{ .HowDecisionsAreMadeWithMultipleAttorneysLink }}#jointly-for-some-decisions-and-jointly-and-severally-attorneys-must-agree-and-act-together-for-some-decisions\" class=\"govuk-link\">attorneys are appointed to act jointly for some decisions, and jointly and severally for others</a>.</p><p class=\"govuk-body\">When there are replacement attorneys, if an attorney dies or can no longer act:</p><ul class=\"govuk-list govuk-list--bullet\"><li>all the replacement attorneys step in and take over making the joint decisions</li><li>the remaining original attorneys will not be able to make the joint decisions anymore</li><li>the replacement and remaining original attorneys can make all other decisions individually</li></ul><h3 class=\"govuk-heading-m\">Examples of restrictions and conditions to change what happens when an attorney who acts jointly for some decisions dies or is unable to act</h3><p class=\"govuk-body\">If the original attorneys are appointed jointly for some decisions, the donor can add restrictions and conditions to change what happens when an original attorney dies or is unable or unwilling to act.</p><p class=\"govuk-body\">Examples:</p><div class=\"govuk-inset-text\"><p class=\"govuk-body\">‘If one of my original attorneys, David Smith or Andrea Williams, is unable or unwilling to act, I then reappoint my remaining original attorney(s), David Smith or Andrea Williams, to continue to make the decisions I have specified to be jointly made.’</p></div><div class=\"govuk-inset-text\"><p class=\"govuk-body\">‘If one of my replacement attorneys, Margaret Taylor or Nick Brown, is unable or unwilling to act, I then reappoint my remaining replacement attorney(s), Margaret Taylor or Nick Brown, to continue to make the decisions I have specified to be jointly made.’</p></div>",
```

Comment:
Not sure what the process is, but thought I should check if anchor links need to be added to the Welsh version of this content too?

Reply:
Good point - I'll replicate the anchors over.